apps/builder/app/shared/copy-paste/plugins/instance.ts

```diff
@@ -1,6 +1,5 @@
 import store from "immerhin";
```

Comment:
plugin-instance.ts?

Reply:
I thought they are all going to be plugins, you think we should repeat the folder name?

Reply:
I mean copy-paste/plugin-instance.ts

Reply:
oh as in instead of folder?

Reply:
Yep